modules/quickstart-public-cloud/pages/installation.adoc

```diff
@@ -1,22 +1,51 @@
 [[quickstart-publiccloud-install]]
 = Installation
 
-Public Cloud providers pre-install {productname}, so you do not need to perform any installation steps.
+Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) model.
+That means that they pre-install {productname}, so you do not need to perform any installation steps.
 However, {productname} Server needs to be registered with {scc} to receive updates before you can sign in.
-For detailed instructions on registering {productname} to {scc}, see xref:installation:server-setup.adoc[].
+
+
+Follow the cloud providers instructions to SSH into the instance.
+
+Run this command to start set up:
+
+----
+yast2 susemanager_setup
+----
+
+Follow the prompts, and wait for the setup to finish.
+
+For detailed instructions on setting up {productname} with {yast}, see xref:installation:server-setup.adoc[].
+
+Register your instance with {scc}, using this command:
+
+----
+sudo SUSEConnect -r <Registration_Number>
+----
+
+You can get your registration number by logging in to {scc} from your browser.
+
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
 
 When you have registered, all {sle} modules will be activated.
-You will also need to activate the Public Cloud module:
+You will also need to activate the Public Cloud module.
+Do this from the command prompt on your instance:
 
-```
-# SUSEConnect -p sle-module-public-cloud/15.1/x86_64
-```
+----
+SUSEConnect -p sle-module-public-cloud/15.1/x86_64
+----
 
-// Check this. LKB 2020-03-06
 For more information on activating public clouds, see xref:installation:pubcloud-setup.adoc[].
 
-A {productname} administrator account is created by default.
-You need this account to access configuration settings in the public cloud provider interface.
+
+Log in to the {productname} {webui} with a web browser, using an address like this:
+
+----
+https://<public_IP>
```

Comment:
actually by IP?  I think we'd better say "by the name that you'll see at the end of the setup routine.  We have this in our default install instruction.

Reply:
Yeah, so the IP is much easier to find in the public cloud interface. I don't recall seeing an actual URL to use, I'll need to run through it again.